VerilogCodingStyle.md

```diff
@@ -766,6 +766,20 @@ always_ff @(posedge clk) begin
 end
 ```
 
+However, keywords that behave more like a function or a task are better aligned
```

Comment:
I frankly don't agree: the guideline is simple enough, and it is best to keep it simple. Notice you are making this even less clear by saying "wait" is an example, as if more exceptions can also be considered.